docs/en/datastores.md

```diff
@@ -0,0 +1,15 @@
+# Data store interfaces
+
+Since the MFA architecture is largely designed to be decoupled, we use a `StoreInterface` implementation to retain
+data between requests. The default implementation for this interface is `SessionStore` which stores data in PHP
```

Comment:
Do you want to mention this uses framework's session API? The session API is also adaptable so if you've got sessions saving on Dynamo you don't need to do anything special. Super minor but potentially saving a developer a few minutes checking that.

Reply:
You probably know more about this than I do - fancy suggesting a change?

Reply:
Just something like:
```
which stores data using SilverStripe's `Session` API provided by framework
```
instead of "PHP sessions". Just in case someone is worried we're circumventing the framework API for some reason.

Reply:
Updated